package.json

```diff
@@ -57,6 +57,7 @@
     "@types/react": "^17.0.40",
     "@types/react-router-dom": "^5.3.3",
     "@types/rimraf": "^3.0.2",
+    "esbuild-loader": "2.16.0",
```

Comment:
We need to pin it for now, since there seems an upstream issue in `esbuild-loader`: https://github.com/facebook/docusaurus/issues/6901